aces/Loris
# [EEG] Sites/Project/subproject filters and permission

This PR:
1. modifies the way project/subproject/site filter are populated. Now those filters are displayed with values the current user has permission for.
3. add some integration tests

## To test
* Visit the module and check that all data entries belong to sites and projects the user has access to.
* Visit the module and check that the site/project/subproject menu filters contain entries the user has access to.

Part of #6829
